src/status/libstatus/accounts/constants.nim

```diff
@@ -178,6 +179,30 @@ var NODE_CONFIG* = %* {
   }
 }
 
-const DATADIR* = "./data/"
-const KEYSTOREDIR* = "./data/keystore/"
-const NOBACKUPDIR* = "./noBackup/"
\ No newline at end of file
+let sep = if defined(windows): "\\" else: "/"
+
+var homeDir = getHomeDir()
+
+var clientDir =
+  if homeDir == "":
+    "."
+  elif defined(macosx):
+    joinPath(homeDir, "Library", "Application Support")
+  elif defined(windows):
+    var targetDir = getEnv("LOCALAPPDATA").string
+    if targetDir == "":
+      joinPath(homeDir, "AppData", "Local")
+    else:
+      targetDir
+  else:
+    var targetDir = getEnv("XDG_CONFIG_HOME").string
+    if targetDir == "":
+      joinPath(homeDir, ".config")
```

Comment:
This is still slightly inconsistent with what I mentioned in the other issue:

`LOCALAPPDATA` and `Library/Application Support` are intended for the "Cache directory" use case, while `XDG_CONFIG_HOME` is intended for the "Roaming profile" use case.

If you wanted this to be a "roaming profile" (i.e. a config directory that should follow the user on every machine), on Windows you should get the `APPDATA` env variable (defaults to `<RootDrive>:\User\<CurrentUser>\AppData\Roaming`). On mac, it should be `Library/Preferences`.

If you wanted it to be "local data" (i.e. arbitrary data specific to the current machine, a cache), then go for `XDG_CACHE_HOME` on Linux (defaults to `joinPath(homeDir, ".cache")`)

Reply:
Thanks. I was aware it's slightly inconsistent, which is the reason for my `can/should refine this at some point` statement in the PR's description.

When we sort out which status-go data should be "local" vs. which should be "roaming" it will be revised for all platorms. In the meantime the directories chosen seemed to me to strike a good balance. For example, I didn't use `XDG_CACHE_HOME` because I didn't want a user to have the impression they could delete `~/.cache/Status` and their account wouldn't be affected (at present the account *would* be deleted).

Reply:
Thanks. I was aware it's slightly inconsistent, which is the reason for my `can/should refine this at some point` statement in the PR's description.

When we sort out which status-go data should be "local" vs. which should be "roaming" it will be revised for all platorms. In the meantime the directories chosen seemed to me to strike a good balance. For example, I didn't use `XDG_CACHE_HOME` because I didn't want a user to have the impression they could delete `~/.cache/Status` and their account wouldn't be affected (at present the account *would* be deleted).